up42/storage.py

```diff
@@ -158,18 +156,13 @@ def _search_stac(
         Search query for storage STAC collection items.
 
         Args:
-            acquired_after: Filter for assets with image acquisition after the datetime or isoformat string e.g.
-                "2022-01-01".
-            acquired_before: Filter for assets with image acquisition before the datetime or isoformat string e.g.
-                "2022-01-30".
-            geometry: Filter for assets intersecting the Polygon geometry. One of FeatureCollection, Feature,
-                dict (geojson geometry), list (bounds coordinates), GeoDataFrame, shapely.Polygon, shapely.Point.
-                All assume EPSG 4326!
-            custom_filter: A CQL2 filter expression for filtering based on properties, see
-                https://pystac-client.readthedocs.io/en/stable/tutorials/cql2-filter.html#CQL2-Filters
+            acquired_after: Search for assets that contain data acquired after the specified timestamp, in `"YYYY-MM-DD"` format.
+            acquired_before: Search for assets that contain data acquired before the specified timestamp, in `"YYYY-MM-DD"` format.
+            geometry: Search for assets that contain STAC items intersecting the provided geometry, in EPSG:4326 (WGS84) format.<br/>For more information on STAC items, see [Introduction to STAC](https://docs.up42.com/developers/api-assets/stac-about).
```

Comment:
This version will be released once the analytics team launch the STAC asset service endpoints. Therefore, it would be OK to add all the future links in this PR. (Not sure if the urls are already available).